### DISCUSSION TITLE
为什么没办法正确下载

### DISCUSSION THREAD

**Discussion by wLLLLLLLuo** [Meloong-Git/PCL#2862] [提问] [answered]
整合包安装失败
→ 解压整合包文件失败
→ 复制文件出错：C:\Users\w\AppData\Local\Temp\PCL\PackInstall\4919\overrides\mods\OpenTerrainGenerator\worlds\BBOP\WorldObjects\Rocks\Stone\patchyrocks\patchyrockfinal\patchyrockfinalC0R0.BO3 → C:\Users\王宇翔\AppData\Local\Temp\fdd037e6-9df9-47aa-8ac3-6a0a94a9bf3b_Release 2.6.10.zip.f3b\.minecraft\versions\Roguelike Adventures and Dungeons\mods\OpenTerrainGenerator\worlds\BBOP\WorldObjects\Rocks\Stone\patchyrocks\patchyrockfinal\patchyrockfinalC0R0.BO3
→ 指定的路径或文件名太长，或者两者都太长。完全限定文件名必须少于 260 个字符，并且目录名必须少于 248 个字符。
   在 PCL.ModBase.CopyFile(String FromPath, String ToPath)
   在 PCL.ModBase.CopyDirectory(String FromPath, String ToPath)
   在 PCL.ModModpack._Closure$__6-0._Lambda$__0(LoaderTask`2 Task)
   在 PCL.ModLoader.LoaderTask`2._Closure$__12-0._Lambda$__0()
   在 PCL.ModBase.CopyFile(String FromPath, String ToPath)
错误类型：System.IO.PathTooLongException

**Comment by MoYuan-CN** [comment]
`指定的路径或文件名太长，或者两者都太长。完全限定文件名必须少于 260 个字符，并且目录名必须少于 248 个字符`
应该是整合包得锅？

**Comment by wLLLLLLLuo** [comment]
我又有了新的问题，整合包下载好了，但是启动的时候崩溃了，没办法正常启动游戏



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年11月9日(周四) 下午2:59
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [Hex-Dragon/PCL2] 为什么没办法正确下载 (Discussion #2862)




 
指定的路径或文件名太长，或者两者都太长。完全限定文件名必须少于 260 个字符，并且目录名必须少于 248 个字符
 应该是整合包得锅？

**Reply by MoYuan-CN** [reply]
游戏崩溃日志传上来谢谢

**Comment by burningtnt** [comment]
上传 mPFL2 导出的错误报告（不要解压，整个上传）

在 2023-11-09 15:04:31，wLLLLLLLuo ***@***.***> 写道：


我又有了新的问题，整合包下载好了，但是启动的时候崩溃了，没办法正常启动游戏



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年11月9日(周四) 下午2:59
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [Hex-Dragon/PCL2] 为什么没办法正确下载 (Discussion #2862)





指定的路径或文件名太长，或者两者都太长。完全限定文件名必须少于 260 个字符，并且目录名必须少于 248 个字符
应该是整合包得锅？

**Reply by MoYuan-CN** [reply]
启动器叫 PCL2（悲
题外话，**非常不建议使用邮箱回复，因为我们可能看到无用的内容！**

**Reply by burningtnt** [reply]
…… 手机回复 + 没有登 GitHub 账户 + 没有 2FA 验证客户端 …… 只能用邮件

**Comment by wLLLLLLLuo** [comment]
[Uploading 错误报告-2023-11-9_14.59.55.zip…]()

**Reply by MoYuan-CN** [reply]
你的日志没传上来.......
拖动文件后，请等待 Uploading xxx 字样消失后再进行发送

**Comment by wLLLLLLLuo** [comment]
[错误报告-2023-11-9_14.59.55.zip](https://github.com/Hex-Dragon/PCL2/files/13305815/-2023-11-9_14.59.55.zip)

**Reply by MoYuan-CN** [reply]
貌似什么东西 SSL 握手失败了
~~本人不太会看崩溃日志，等等别人吧~~

**Reply by burningtnt** [reply]
卸载你现在的 Java，安装这个 Java 8：https://github.com/bell-sw/Liberica/releases/download/8u392%2B9/bellsoft-jre8u392+9-windows-amd64-full.msi

**Comment by wLLLLLLLuo** [comment]
感谢



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年11月9日(周四) 下午4:51
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [Hex-Dragon/PCL2] 为什么没办法正确下载 (Discussion #2862)




 
貌似什么东西 SSL 握手失败了
 本人不太会看崩溃日志，等等别人吧

**Comment by wLLLLLLLuo** [comment]
> 卸载你现在的 Java，安装这个 Java 8：https://github.com/bell-sw/Liberica/releases/download/8u392%2B9/bellsoft-jre8u392+9-windows-amd64-full.msi
好的，我试一下

**Comment by wLLLLLLLuo** [comment]
> 卸载你现在的 Java，安装这个 Java 8：https://github.com/bell-sw/Liberica/releases/download/8u392%2B9/bellsoft-jre8u392+9-windows-amd64-full.msi
仍然不能启动游戏
[错误报告-2023-11-9_18.59.41.zip](https://github.com/Hex-Dragon/PCL2/files/13307238/-2023-11-9_18.59.41.zip)

**Reply by burningtnt** [reply, marked as answer]
日志显示你仍然在使用原来的 Java 启动游戏
请先安装新的 Java，然后前往 PCL2 特定游戏设置中将 Java 路径改为 JRE 8.392, Bellsoft

**Comment by wLLLLLLLuo** [comment]
> 日志显示你仍然在使用原来的 Java 启动游戏 请先安装新的 Java，然后前往 PCL2 特定游戏设置中将 Java 路径改为 JRE 8.392, Bellsoft
感谢您的帮助，我已经成功启动了游戏
